lib/model.js

```diff
@@ -2122,6 +2128,41 @@ Model.watch = function(pipeline, options) {
   return new ChangeStream(this, pipeline, options);
 };
 
+/**
+ * _Requires MongoDB >= 3.6.0._ Starts a [MongoDB session](https://docs.mongodb.com/manual/release-notes/3.6/#client-sessions)
+ * for causal consistency in reads.
```

Comment:
I think probably better to leave it up to users to read up on why session are beneficial. It's not just for causal, but also retryable writes and eventually transactions (likely other features).